.fixtures.yml

```diff
@@ -3,12 +3,8 @@ fixtures:
     "augeas_core":
       "repo": "puppetlabs/augeas_core"
       "puppet_version": ">= 6.0.0"
-    "mailalias_core":
-      "repo": "puppetlabs/mailalias_core"
-      "puppet_version": ">= 6.0.0"
```

Comment:
Since https://github.com/puppetlabs/puppetlabs-mailalias_core/pull/14 there is no longer a need to restrict this module to only puppet 6.

Reply:
I think it'd still be better to restrict it since the version of the core module shipped in Puppet 5 might differ eventually.